pkg/csi_driver/controller.go

```diff
@@ -101,15 +106,61 @@ func (s *controllerServer) CreateVolume(ctx context.Context, req *csi.CreateVolu
 			return nil, status.Error(codes.AlreadyExists, err.Error())
 		}
 	} else {
+
+		// Get the reserved-IPV4-cidr from the request. It was not added in the service instance as it is only consumed by the controller
+		reservedIPV4CIDR, reservedIPV4CIDRPresent := req.GetParameters()[paramReservedIPV4CIDR]
+		if !reservedIPV4CIDRPresent {
+			return nil, fmt.Errorf("The required parameter reserved-ipv4-cidr was not provided")
```

Comment:
This is not a required parameter.  If it was not specified, then we leave ReservedIpRange empty

Reply:
Ok. What should be used as a reservedIPRange in that case?